save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/entities/benchmarks/TabMenubar.kt

```diff
@@ -0,0 +1,64 @@
+package com.saveourtool.save.entities.benchmarks
+
+import com.saveourtool.save.domain.Role
+
+/**
+ * Interface for tab bar in many pages
+ */
+interface TabMenubar<T> {
```

Comment:
```suggestion
interface TabMenubar<T : Enum<T>> {
```
Can we add such constraint here? Or is it applicable for inline functions only?

Reply:
@Arrgentum what about this?